Branch-SDK/Branch-SDK/BranchUniversalObject.m

```diff
@@ -132,6 +132,16 @@ - (void)showShareSheetWithLinkProperties:(BranchLinkProperties *)linkProperties
     }
     UIActivityViewController *shareViewController = [[UIActivityViewController alloc] initWithActivityItems:items applicationActivities:nil];
     
+    if ([shareViewController respondsToSelector:@selector(completionWithItemsHandler)]) {
+        shareViewController.completionWithItemsHandler = ^(NSString *activityType, BOOL completed, NSArray *returnedItems, NSError *activityError) {
```

Comment:
would this still work on iOS 6 though if you have explicitly listed the newer completion handler format on this line? if the goal is backwards compatibility, is this backward compatible?